rest/src/test/java/com/ericsson/bss/cassandra/ecchronos/rest/TestRepairSchedulerRESTImpl.java

```diff
@@ -86,7 +87,8 @@ public void testListEntry()
         assertThat(scheduledRepairJob.table).isEqualTo("tb");
         assertThat(scheduledRepairJob.lastRepairedAtInMs).isEqualTo(expectedLastRepairedAt);
         assertThat(scheduledRepairJob.repairedRatio).isEqualTo(0.0d);
-        assertThat(scheduledRepairJob.repairIntervalInMs).isEqualTo(expectedRepairInterval);
+        assertThat(scheduledRepairJob.nextRepairInMs).isEqualTo(expectedLastRepairedAt + repairInterval);
+        assertThat(scheduledRepairJob.status).isEqualTo(Status.ERROR);
```

Comment:
Nit:

With the Status column it might be wort changing expectedLastRepairedAt/repairInterval to test different statuses.
But this should already be tested with CompletedRepairJob/ScheduledRepairJob

Reply:
Yes, it is tested elsewhere. 
But i updated 1 test case so that other values than status=ERROR and ratio=0.0 is tested